### ISSUE TITLE
About accuracy of pretrained  vgg16.npy

### ISSUE THREAD

**Issue by hjh3169** [SidHard/tfAlexNet#1]
I downloaded the pretrained model vgg16.npy and tested it on the ImageNet validation set. However I found that the accuracy is not as high as that given on Caffe. I only get 66.9% of Top-1 accuracy...
